### ISSUE TITLE
Delegating prefix subset "SystemWrapper.Core" on Nuget for a netstandard 2.0 version

### ISSUE THREAD

**Issue by mperdeck** [jozefizso/SystemWrapper#59]
Thanks for your SystemWrapper.Wrappers package, it is exactly what I needed to mock the file system in my unit tests. Except that my project is in .Net Core 3.1. So I ported your package to netstandard 2.0.

I would like to publish this package on Nuget under the package id "SystemWrapper.Core". However, I think you have the package ID prefix "SystemWrapper" reserved with Nuget. So I was not able to push my package.

Would you be willing please to delegate the prefix subset "SystemWrapper.Core" to me? That would allow me to push my package with an id that makes it clear it is a version of your work.

**Comment by jozefizso** [comment]
Hi @mperdeck, are your projects targeting netstandard2.0 mergeable into the master of SystemWrapper?

I need to check how prefixes can be delegated in nuget.

**Comment by mperdeck** [comment]
No, they are not. I think it would be painful to try to maintain a single multi target version of SystemWrapper, because some System  methods are only available in .Net Framework, while others are only available in .Net Core. 

My idea at the moment is that there will be 2 editions - SystemWrapper for .Net Framework maintained by you, and SystemWrapper.Core  for .Net Core. maintained by myself.

**Comment by jozefizso** [comment]
I think this is the repo with new code https://github.com/mperdeck/SystemWrapper.Core, am I right?

One issue for me is that the license was changed from **Microsoft Public License (MS-PL)** to **MIT License** and the code is not attributed to the original authors.

I think projects in current repo can be retargeted to **netstandard2.0** and the code can be conditionally compiled for both .NET Framework and .NET Core, although I don't have time to do it myself at the time.

If **SystemWrapper.Core** was a proper fork of the **SystemWrapper** project with prospect of merging the new code into original code base I won't have issues with forking the project or having a new package for the new code.

https://guides.github.com/activities/forking/

**Comment by hmartinez82** [comment]
@jozefizso Me comment in #61 was too soon. I can totally take care of this. We would have to convert the .csproj files to the new SDK-Style format first. Then we can multitarget (i.e. target .NetFramework 4.5 and .Net Standard) in the same package. I'll have a shot in the conversion soon.
